CODE_OF_CONDUCT.md

```
@@ -0,0 +1,58 @@
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the project maintainers at [INSERT EMAIL].
+All complaints will be reviewed and investigated promptly and fairly.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)